src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> comparator = new Comparator<>() {
+            @Override
+            public int compare(Employee employee2, Employee employee1) {
```

Comment:
```suggestion
            public int compare(Employee first, Employee second) {
```
second employee being the first parameter is a bit confusing